README.md

```diff
@@ -55,40 +52,44 @@ of the [Federated Identity Community Group](https://fedidcg.github.io/).
 
 ## Introduction
 
-The goal of this project is to provide a purpose-built API for enabling secure and user-mediated access to cross-site top-level unpartitioned cookies. 
-This is accomplished with integration with the [Credential Management API](https://w3c.github.io/webappsec-credential-management/) and the [Login Status API](https://w3c-fedid.github.io/login-status/#introduction) to enable easy integration with alternative authentication mechanisms.
-A site that wants a user to log in calls the `navigator.credentials.get()` function with arguments defined in this spec. The browser ensures there is appropriate user mediation and identity provider opt-in. With those assurances, the browser may also decide there is no additional privacy loss associated with access to unpartitioned state, and choose to automatically grant access to Storage Access requests.
+The goal of this proposal is to create more flexibility for Identity Providers (IdPs) who wish to integrate with Federated Credential Management and the gain the benefits of more contextually-relevant user mediated access to cross-site top-level unpartitioned cookies.
 
-It is possible to look at this proposal as an “axiomatic base” of FedCM; simplifying FedCM down to its barest essentials that will allow it to still function in a way that provides value to both users and identity providers. One benefit of this approach is it allows for incremental adoption of full FedCM by existing identity providers. It defines useful user-agent behaviors for FedCM when some of the specified endpoints are not defined. This also has some implementation benefits for user-agent developers, as it does not require that an entirely parallel codepath be developed, tested, and maintained.
+This is accomplished with three proposed changes to the [FedCM specification](https://w3c-fedid.github.io/FedCM/):
 
+* Allow IdPs to store display information about users accounts up-front via the [Login Status API](https://w3c-fedid.github.io/login-status/#introduction), which we will refer to as "FedCM Accounts Push."
+* Allows IdPs to skip implementation of the `client_metadata_endpoint`, `id_assertion_endpoint`, `accounts_endpoint`, and the `login_url` by defining user agent semantics when any of these are not present; this will be referred to as "FedCM Optional Endpoints."
+* Allow IdPs to store endpoint configuration information up-front via the Login Status API; this will be referred to as "FedCM Config Push."
```

Comment:
```suggestion
* Allow IdPs to store display information about user accounts up-front via the [Login Status API](https://w3c-fedid.github.io/login-status/#introduction), which we will refer to as "FedCM Accounts Push."
* Allows IdPs to skip implementation of the `client_metadata_endpoint`, `id_assertion_endpoint`, `accounts_endpoint`, and `login_url` by defining user agent semantics when any of these are not present; these will be referred to as "FedCM Optional Endpoints."
* Allow IdPs to store endpoint configuration information up-front via the Login Status API; this will be referred to as "FedCM Config Push."
```